skycoin/skywire
# Improvements for the VPN client

Did you run `make format && make check`?
The go code was not changed. `npm run lint` and `npm run build` were used.

 Changes:	
- Now the sevcer list is populated with data obtained from the discovery service. All columns not returned by the discovery service were removed from the UI. The code related to the removed columns and not longer used was not removed, but commented, just in case it is needed in the future.

- A bug which made the paginator of the server list page work incorrectly was solved.

- Now the countries are sorted by country name, not by country code. The previous method had problems with some countries and the list appeared to be wrongly sorted.

- Now the correct value of the `killswitch` property is shown. Related to https://github.com/skycoin/skywire/pull/720

How to test this PR:
For checking the server list, use a web browser to open `{SkywireManagerUrl}/#/vpn/{visorPk}/servers/public/1`. That will show the list with the servers obtained from the discovery service. Use the paginator at the bottom of the page and check if it works well. Also, pressing the flag icon at the top of the country column should order the server by country name.

For checking the value of the `killswitch` property, go to the settings tab.

NOTE: this PR replaces https://github.com/skycoin/skywire/pull/782
